Effect for Complete Beginners/10. Introduction to Fibers.md

```diff
@@ -152,47 +143,42 @@ As you can see, both must receive an `Effect`, and they both return an `Effect`
 One advantage of receiving the fiber upon creation is the ability to interrupt it later:
 
 ```ts
-import { Effect, pipe, Fiber, Schedule, Duration } from "effect";
+import { Duration, Effect, Fiber, Schedule } from "effect";
 
 const program = Effect.gen(function* () {
-  const forkedTask = yield* Effect.forkDaemon(
-    pipe(
-      Effect.log("Called"),
-      Effect.repeat({
-        schedule: Schedule.spaced(Duration.seconds(0.5)),
-        times: 100,
-      })
-    )
+  const forkedTask = yield* Effect.log("Called").pipe(
+    Effect.repeat({
+      schedule: Schedule.spaced(Duration.seconds(0.5)),
+      times: 100,
+    }),
+    Effect.onInterrupt(() => Effect.log("Interrupted fiber")),
```

Comment:
It may be a bit better to add an `Effect.onInterrupt` to the interrupted fiber to display the log message. Then, the reader can clearly see the message originated from `fiber=#1`.